.github/workflows/ci.yml

```diff
@@ -66,6 +66,12 @@ jobs:
           - toxenv: test-opencv-xdist
             os: ubuntu-latest
             python-version: '3.10'
+          - toxenv: test-devdeps-xdist
+            os: ubuntu-latest
+            python-version: '3.10'
+          - toxenv: test-devdeps-xdist
+            os: macos-latest
+            python-version: '3.10'
```

Comment:
Probably not needed to run devdeps jobs for every pull request in ci.yml.  There's a ci_cron.yml workflow here that runs the devdeps jobs every Monday morning that is probably more appropriate for this.  And feel free to run it more often if weekly is not enough to catch these issues.

Reply:
This is up to @zacharyburnett, as they added those changes. My original intent was to just test these as part of our weekly regression tests.